conf/messages.en

```diff
@@ -18,6 +18,7 @@ site.canInclude.individual = You can include:
 site.canInclude.agent = Your client can include:
 site.cannotInclude.individual = You cannot include:
 site.cannotInclude.agent = Your client cannot include:
+site.selectAllTheApply = Select all that apply.
```

Comment:
Should we have this named as `site.selectAllThatApply` instead?

Reply:
thanks, typo